### DISCUSSION TITLE
Nominate Anthony Dahanne as a content maintainer

### DISCUSSION THREAD

**Discussion by anthonydahanne** [paketo-buildpacks/feedback#293] [Content Team]
Hey https://github.com/orgs/paketo-buildpacks/teams/content-maintainers, I'd like to propose to promote myself, @anthonydahanne  as a content maintainer. I believe @ForestEckhardt  is now the only active maintainer, and I proved myself involved with content several times with the [samples project](https://github.com/paketo-buildpacks/samples/pulls?q=is%3Apr+author%3Aanthonydahanne+is%3Aclosed), as well as with [documentation](https://github.com/paketo-buildpacks/paketo-website/pulls?q=is%3Apr+author%3Aanthonydahanne+is%3Aclosed) and [blog posts](https://github.com/paketo-buildpacks/blog/pulls?q=is%3Apr+author%3Aanthonydahanne+is%3Aclosed).

@paketo-buildpacks/content-maintainers Can I get a 👍 / 👎 vote?

Thank you!

**Comment by ForestEckhardt** [comment]
I think that there are currently enough votes to codify this. Welcome to the content maintainer team!

**Reply by anthonydahanne** [reply]
thank you very much!
